docs/user-guide/cli-install-configure-zosmf-security.md

```diff
@@ -1,21 +1,27 @@
 # Configuring z/OSMF Security
 
-Review the tasks that security administrators must complete to configure z/OSMF security for your Zowe CLI implementation.
+Review the tasks that security administrators must complete to configure z/OSMF security for your implementation of Zowe client-side components. **(Is "Zowe client-side components" correct?)**
 
 :::info Required role: security administrator
 :::
 
 ## Configuring z/OS REST services SAF security
 
-A security administrator must configure security to allow z/OSMF System Authorization Facility (SAF) access to the resources that Zowe CLI requires. Zowe CLI uses REST endpoints that are associated with each z/OSMF REST API. After you complete all z/OSMF and z/OSMF cloud provisioning configurations, you can test your connection to z/OSMF to verify that Zowe CLI can communicate with z/OS systems.
+:::note
+
+If you are connecting to the mainframe with other methods than a z/OSMF profile, you do not need to configure z/OSMF security. Other connection options might include using FTP, or your custom API.
+
+:::
+
+A security administrator must configure security to allow z/OSMF System Authorization Facility (SAF) access to the resources that Zowe client-side components require. Zowe client-side components use REST endpoints that are associated with each z/OSMF REST API. After you complete all z/OSMF and z/OSMF cloud provisioning configurations, you can test your connection to z/OSMF to verify that your Zowe client-side components can communicate with z/OS systems.
 
 :::caution
 
-Before you allow users to issue z/OS console commands with Zowe CLI, security administrators should ensure that they provide access to commands that are appropriate for their organization.
+Before you allow users to issue z/OS console commands with Zowe client-side components, security administrators should ensure that they provide access to commands that are appropriate for their organization.
 
 :::
 
-The following table details the required z/OSMF REST services and examples of the Zowe CLI features they enable.
+The following table details the required z/OSMF REST services and examples of the Zowe CLI features **(are these features applicable to only Zowe CLI? If so, should we add other examples applicable to other components? Or are these features available across all client-side components?)** they enable.
```

Comment:
@adam-wolfe please respond

Reply:
Cloud provisioning services and z/OSMF workflow services are specific to Zowe CLI.